packages/storybook-react/src/stories/Select/argTypes.ts

```diff
@@ -0,0 +1,49 @@
+export const argTypes = {
+  disabled: {
+    name: 'disabled',
+    type: { name: 'boolean', required: false },
+    table: {
+      category: 'HTML attribute',
+      defaultValue: { summary: false },
+    },
+  },
+  invalid: {
+    name: 'invalid',
+    type: { name: 'boolean', required: false },
+    table: {
+      defaultValue: { summary: false },
+    },
+  },
+  required: {
+    name: 'required',
+    type: { name: 'boolean', required: false },
+    table: {
+      category: 'HTML attribute',
+      defaultValue: { summary: false },
+    },
+  },
+  noscript: {
+    name: 'noscript',
+    type: { name: 'boolean', required: false },
+    table: {
+      category: 'HTML attribute',
+      defaultValue: { summary: false },
+    },
+  },
+  value: {
+    name: 'value',
+    type: { name: 'string' },
+    table: {
+      category: 'HTML attribute',
+      defaultValue: { summary: '' },
+    },
+  },
+};
+
+export const options = [
+  { value: '', label: 'Select an option', disabled: true },
```

Comment:
Maybe we should investigate the placeholder option some more, for a separate PR.

Would be nice to be able to do `<Select placeholder="Select an option">`. The placeholder option should ideally use a `placeholder-color` design token instead of `disabled-color`.

So if the placeholder attribute could render the following it could be beneficial:

```html
<select class="utrecht-select utrecht-select--placeholder-selected">
  <option value="" disabled selected class="utrecht-select__option utrecht-select__option--placeholder">Placeholder text</option>
</select>
```